### ISSUE TITLE
[NOT ISSUE] This project

### ISSUE THREAD

**Issue by Praymundo** [Praymundo/NES-Emulator-Web#2]
The primary goal of this project is to build an NES Emulator using web technologies.
I don't expect this to be perfect or fast as it could be using other technologies by the simple fact that we are bound to the browser.
I hope it can be a fun project.

I think that we can focus on just the processor and its instructions for the moment.
It is painfully slow now, but I'm trying to improve the execution speed without changing too much how the real processor works.

# Workflow

To avoid conflicts and to keep the project under control I'm requiring any change to be done via Pull Requests (can be from a branch or from a fork), but any change will be reviewed before applied to the master.
Remember that this is a project to learn as well, so I want to understand everything that is being applied. 

# Technologies

As stated before any web technology is acceptable, however I would like to avoid any server side for now.
In other words anything that runs on the browser itself.

# Code quality

I would like everyone to use the docker container provided and VSCode as IDE.
However it is not possible, people may have different tastes, so the only thing I'm going to require is the "Prettier" formatter for the code, in this way we avoid discussions about code style (I myself don't agree with everything there, but the advantages worth the sacrifice).

# Next steps

Next steps or tasks will be kept as issues that can assigned to people and closed with pull requests.

# This issue

I'm going to update this issue as required with new rules.
We may create the default **contribution** file at some point as well, but this will do for now.
